cytosnake/guards/path_guards.py

```diff
@@ -0,0 +1,29 @@
+"""
+Module: path_guards.py
+
+
+Contains functions that ensures that types that are considered paths are
+checked. 
```

Comment:
```suggestion
Functions for checking paths. 
```

Does this still capture your meaning? I don't want to simplify too much!